src/components/AccountDetails/index.tsx

```diff
@@ -267,6 +268,7 @@ export default function AccountDetails({
       if (isXDEFI) name = SUPPORTED_WALLETS.XDEFI.name
       else if (isTalisman) name = SUPPORTED_WALLETS.TALISMAN.name
       else if (isRabby) name = SUPPORTED_WALLETS.RABBY.name
+      else if (isBitKeep) name = SUPPORTED_WALLETS.BITKEEP.name
       else if (isCoinbase) name = SUPPORTED_WALLETS.WALLET_LINK.name
       // metamask as last check, because most of the wallets above are likely to set isMetaMask to true too
       else if (isMetaMask) name = SUPPORTED_WALLETS.METAMASK.name
```

Comment:
Need to add condition for bitKeep to show status icon 

also in Web3Status file 
pseudo code
 
```
else if (connector === bitkeep) {
      return (
        <IconWrapper size={16}>
          <img src={BitkeepIcon} alt={'Bitkeep logo'} />
        </IconWrapper>
      )
}
```

Reply:
can't we instead add a default icon status with Identity and use that for bitkeep?
```js
function getStatusIcon() {
    if (connector === walletlink) {
      return (
        <IconWrapper size={16}>
          <img src={CoinbaseWalletIcon} alt={'Coinbase Wallet logo'} />
        </IconWrapper>
      )
    } else if (connector === walletconnect) {
      return (
        <IconWrapper size={16}>
          <img src={WalletConnectIcon} alt={'Wallet Connect logo'} />
        </IconWrapper>
      )
    } else if (connector === gnosisSafe) {
      return (
        <IconWrapper size={16}>
          <img src={GnosisSafeIcon} alt={'Gnosis Safe logo'} />
        </IconWrapper>
      )
    } else if (connector === near) {
      return (
        <IconWrapper size={16}>
          <img src={NearIcon} alt={'Near Wallet'} />
        </IconWrapper>
      )
    }
    return (
      <IconWrapper size={16}>
        <Identicon />
      </IconWrapper>
    )
  }
  ```